docs/appendices/release-notes/5.5.3.rst

```diff
@@ -49,4 +49,12 @@ See the :ref:`version_5.5.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed a regression introduced in 5.3.0 that prevented the evaluation of
```

Comment:
What is the issue in this case? failure? or wrong value inserted in the replicas?

Reply:
wrong value, it was NULL on the whole object. I took change entry from the referenced commit, since it's same issue but with different angle (gen expressions this time)

Reply:
I think it's better if we describe the user facing behaviour, as `prevented the evaluation...` doesn't make it clear of what was happening.

Reply:
Rephrased in https://github.com/crate/crate/pull/15272/commits/adcd6315ef441419d00f59a91faa6431c3cf5c30

Reply:
thx!